7.figures/nbconverted/Figure3_model_evaluation.r

```diff
@@ -0,0 +1,212 @@
+suppressPackageStartupMessages(library(dplyr))
+suppressPackageStartupMessages(library(ggplot2))
+suppressPackageStartupMessages(library(patchwork))
+
+# Load variables important for plotting (e.g., themes, phenotypes, etc.)
+source("themes.r")
+
+figure_dir <- "figures"
+output_main_figure_3 <- file.path(figure_dir, "main_figure_3_multiclass_model_eval.png")
+output_sup_fig_corr <- file.path(figure_dir, "supplementary", "supplementary_pairwise_correlations.png")
+
+# Load confustion matrix data
+cm_dir <- file.path("../3.evaluate_model/evaluations/confusion_matrices")
+cm_file <- file.path(cm_dir, "compiled_cm_data.tsv")
+
+cm_df <- readr::read_tsv(
+    cm_file,
+    col_types = readr::cols(
+        .default = "c",
+        shuffled = "l",
+        Count = "d"
+    )
+) %>%
+    dplyr::select(!...1) %>%
+    dplyr::group_by(True_Label, data_split, shuffled, feature_type) %>%
+    dplyr::mutate(
+        total_count = sum(Count),
+        ratio = Count / total_count
+    )
+
+# Reverse order of predicted label for plotting
+cm_df$Predicted_Label <-
+    factor(cm_df$Predicted_Label, levels = rev(unique(cm_df$Predicted_Label)))
+
+# Order feature types for plotting
+cm_df$feature_type <-
+    dplyr::recode_factor(cm_df$feature_type, !!!facet_labels)
+
+head(cm_df)
+
+custom_labeller <- function(value) {
```

Comment:
Is it required that you need to make this custom labeller or could you just label each row as "final" or "shuffled"?

Reply:
Great question! Labeling each row is one way to do it. Is this what you would prefer? To me, `Shuffled: FALSE` is more clear than `Final`, but it is definitely clunkier :/

Reply:
I can see both sides! I think I will default to what you think since you have more experience with how readers interpret figures. Shuffled: FALSE is still very clear and I think that someone can put 2 and 2 together to know that represents the final model.